modules/nf-core/fastqc/main.nf

```diff
@@ -2,10 +2,10 @@ process FASTQC {
     tag "$meta.id"
     label 'process_medium'
 
-    conda "bioconda::fastqc=0.11.9"
+    conda "bioconda::fastqc=0.12.1"
```

Comment:
Can you please make a separate PR for the changes to `fastqc`?

Reply:
How can I remove this from the current PR?
I created a PR for my branch "bedtoolsShift" and it was merged into this PR

Reply:
You can undo those commits normally :) Otherwise you can just copy the code from `master` into those files and it should be fixed too

Reply:
done